Integrations/ZeroFox/ZeroFox.py

```diff
@@ -0,0 +1,510 @@
+        'PerpetratorURL': alert.get('perpetrator', {}).get('url'),
+        'PerpetratorTimeStamp': alert.get('perpetrator', {}).get('timestamp'),
+        'PerpetratorType': alert.get('perpetrator', {}).get('type'),
+        'PerpetratorID': alert.get('perpetrator', {}).get('id'),
+        'PerpetratorNetwork': alert.get('perpetrator', {}).get('network'),
+        'RuleGroupID': alert.get('rule_group_id'),
+        'AssetID': alert.get('asset', {}).get('id'),
+        'AssetName': alert.get('asset', {}).get('name'),
+        'AssetImage': alert.get('asset', {}).get('image'),
+        'Status': alert.get('status'),
+        'Timestamp': alert.get('timestamp'),
+        'RuleName': alert.get('rule_name'),
+        'LastModified': alert.get('last_modified'),
+        'ProtectedLocations': alert.get('protected_locations'),
+        'DarkwebTerm': alert.get('darkweb_Term'),
+        'Reviewed': alert.get('reviewed'),
+        'Escalated': alert.get('escalated'),
+        'Network': alert.get('network'),
+        'ProtectedSocialObject': alert.get('protected_social_object'),
+        'Notes': alert.get('notes'),
+        'RuleID': alert.get('rule_id'),
+        'EntityAccount': alert.get('entity_account'),
+        'Tags': alert.get('tags')
+    }
+
+
+def get_entity_contents_war_room(contents):
+    return {
+        'Name': contents.get('Name'),
+        'Type': contents.get('Type'),
+        'Policy': contents.get('Policy'),
+        'Email': contents.get('EmailAddress'),
+        'Tags': contents.get('Labels'),
+        'ID': contents.get('ID')
+    }
+
+
+def get_entity_contents(entity):
+    return {
+        'ID': entity.get('id'),
+        'Name': entity.get('name'),
+        'EmailAddress': entity.get('email_address'),
+        'Organization': entity.get('organization'),
+        'Labels': entity.get('labels'),
+        'StrictNameMatching': entity.get('strict_name_matching'),
+        'PolicyID': entity.get('policy_id'),
+        'Profile': entity.get('profile'),
+        'EntityGroupID': entity.get('entity_group', {}).get('id'),
+        'EntityGroupName': entity.get('entity_group', {}).get('name'),
+        'EntityTypeID': entity.get('type', {}).get('id'),
+        'EntityTypeName': entity.get('name', {}).get('name')
+    }
+
+
+# returns the convention for the war room
+def get_alert_contents_war_room(contents):
+    return {
+        'ID': contents.get('ID'),
+        'Protected Entity': contents.get('EntityName', '').title(),
+        'Content Type': contents.get('AlertType', '').title(),
+        'Alert Date': contents.get('Timestamp', ''),
+        'Status': contents.get('Status', '').title(),
+        'Source': contents.get('Network', '').title(),
+        'Rule': contents.get('RuleName'),
+        'Severity': contents.get('Severity'),
+        'Notes': contents.get('Notes') if contents.get('Notes') != '' else None,
+        'Tags': contents.get('Tags')
+    }
+
+def get_authorization_token():
```

Comment:
this should return the token
use integration context to store the token

Reply:
TODO: use integration context to store the token